hinosxz/chowchow
# ✨ Add UserAlertManager to handle liked shows VS shows on watch list

## What does this PR do?
It add some an alert manager to handle the alerts of a user. It's not a controller, it does not retrieve he alerts but manage a list of alerts in order to return them if the related show is rated by the user

## Motivation (link to Trello card)
https://trello.com/c/cjtIa0lU/61-add-alerting-logic

## Testing Guidelines
Go on [staging](https://trello.com/c/cjtIa0lU/61-add-alerting-logic)
- Search "Rick and morty" (this show has upcoming episodes)
- Add it to your watch list
- Update the page, no alerts should be displayed.
- Go to your watch list and rate the show
- Go back to home, an alert is here !
## Release Checklist

-   [ ] This code has automated tests
-   [ ] This code has documentation

## Additional Notes
